### ISSUE TITLE
Overview dashboard's "Manager" billboard says "Online Online" when scylla_manager_servers.yml has the new port

### ISSUE THREAD

**Issue by ziggythehamster** [scylladb/scylla-monitoring#1121]
In Scylla Monitoring 3.5, due to the change introduced in #1014, if you have the new port configured, you end up with two jobs, `scylla_manager` and `scylla_manager1`. They both report the same thing because they both point at :5090. The query doesn't do any aggregation or filtering to prevent this from displaying:

![image](https://user-images.githubusercontent.com/66674/98614851-224c1f00-22ae-11eb-8bab-6c16d17ae2ad.png)

**Comment by ziggythehamster** [comment]
Changing the query to this seems to fix it:

```
(1+sum(scylla_manager_task_active_count{type=~"repair",cluster=~"$cluster|$^"}) + sum(scylla_manager_task_active_count{type=~"backup",cluster=~"$cluster|$^"}) * 2) or (1+max(scylla_manager_server_current_version))
```

But I have no experience with grafana to know if that's really the right thing to do here or not. The regex anchors being backwards is also super odd to me, but maybe it's supposed to be that way

**Comment by amnonh** [comment]
Should be fixed by #1118

**Comment by amnonh** [comment]
Closing as duplicate of #1109 and the fix in  #1118 re-open if you are still seeing issues after those fixes

**Comment by ziggythehamster** [comment]
This happens *with* #1118's patch applied. [If you look at this comment](https://github.com/scylladb/scylla-monitoring/issues/1121#issuecomment-724386632), you would be able to see the query being used is the one from #1118.
